internal/servers/private_projects_server.go

```
@@ -129,6 +132,43 @@ func (s *PrivateProjectsServer) Get(ctx context.Context,
 
 func (s *PrivateProjectsServer) Create(ctx context.Context,
 	request *privatev1.ProjectsCreateRequest) (response *privatev1.ProjectsCreateResponse, err error) {
+	// To avoid potential issues with the lenght of full project names we limit the number of segments of a
+	// full project path.
+	const max = 4
```

golangci-lint (2.12.2)

[error] const max has same name as predeclared identifier

(predeclared)